server/src/api/FriendRoutes.ts

```diff
@@ -1,26 +1,132 @@
 import express from "express";
 import { authenticateToken } from "../middlewares";
 import { prisma } from "../lib/prisma";
-import { OK_CODE } from "../lib/StatusCodes";
+import {
+  OK_CODE,
+  INTERNAL_ERROR_CODE,
+  NOT_FOUND_CODE,
+  INVALID_REQUEST_CODE,
+} from "../lib/StatusCodes";
 
 const router = express.Router();
 
-router.get("", authenticateToken, async (req, res) => {
-  const userId = req.body.user.id;
+router.get("/:id", authenticateToken, async (req, res) => {
+  const userId = Number(req.params.id);
```

Comment:
why was this switched to be from params id?

Reply:
resolved to body.user.id for all